### PR TITLE
feat: Add tags filter for go serve tab

### PR DESCRIPTION
**Pull request by tomthornton**
As of right now, no content items are tagged so filtering won't show any content items. We can let them add the tags to the content items in rock.
Screen Shot 2021-11-15 at 12 08 35 PM

### CODE REVIEW

File: apollos-church-api/package.json [hunk 1]
```diff
@@ -27,11 +27,13 @@
     "@apollosproject/data-connector-cloudinary": "^2.34.0",
     "@apollosproject/data-connector-onesignal": "^2.34.0",
     "@apollosproject/data-connector-passes": "^2.34.0",
+<<<<<<< go-serve-tag-filter
```
Comment (nlewis84) [inline apollos-church-api/package.json:30]: Can delete this

File: apollos-church-api/package.json [hunk 2]
```diff
@@ -27,11 +27,13 @@
...
+    "@apollosproject/data-connector-postgres": "^2.37.1-canary.2",
```
Comment (nlewis84) [inline apollos-church-api/package.json:31]: Merge conflict with this line and line 32

File: apollos-church-api/package.json (hunk 1 above)
Comment (tomthornton) [reply apollos-church-api/package.json]: Whoops you're right

**Review by nlewis84 — APPROVED** [review]
Looks good when I run it on my machine.

**Pull request merged**